### ISSUE TITLE
adorn_rounding does not work on 3-way tabyls

### ISSUE THREAD

**Issue by hase3** [sfirke/janitor#347]
## Bug reports

adorn_rounding does not work on 3-way tabyls

small sample is as follows.

```r
mtcars %>%
  tabyl(am, cyl, vs) %>%
  adorn_percentages("col")  %>%
  adorn_rounding(digits=2)
```

**Comment by sfirke** [comment]
When I run that I get this:
```
library(janitor); mtcars %>%
    tabyl(am, cyl, vs) %>%
    adorn_percentages("col")  %>%
    adorn_rounding(digits=2)


#> $`0`
#>  am 4 6         8
#>   0 0 0 0.8571429
#>   1 1 1 0.1428571
#> 
#> $`1`
#>  am   4 6   8
#>   0 0.3 1 NaN
#>   1 0.7 0 NaN
```
Does that happen for you too?  What part would you say does not work?

The `NaN` is intended behavior, there are no combinations of cyl = 8 and vs = 1 in the data so you get `NaN` for percentages of NA.  The next step, `adorn_pct_formatting` turns those into a better-looking `"-"`.

**Comment by sfirke** [comment]
I deleted my initial response to you, it was entirely wrong.  Thanks for this bug report!  This is a huge hole in the package and I can't believe I missed it for this long.  I am working on a big overhaul of the `adorn_` functions right now so great timing for me to patch this.

**Comment by hase3** [comment]
Thank you @sfirke !

My first report would be too simple to prevent misunderstandings.
I'm convinced that you understood my intent exactly, but just in case, here is a more accurate reports as follows:

```r
# Actual results
library(janitor); mtcars %>%
  tabyl(am, cyl, vs) %>%
  adorn_percentages("col")  %>%
  adorn_rounding(digits=2)

# Expected results
tabyl(subset(mtcars,vs==0),am,cyl) %>%
    adorn_percentages("col")  %>%
        adorn_rounding(digits=2)
tabyl(subset(mtcars,vs==1),am,cyl) %>%
    adorn_percentages("col")  %>%
        adorn_rounding(digits=2)
```

**Comment by sfirke** [comment]
I just closed this by merging #355, not sure why it didn't auto-close but this should work now.  Does  this work for you now?

**Comment by hase3** [comment]
@sfirke
Sorry for late reply.
I tried ‘janitor’ version 2.0.0 install via github.
This version seems working fine for my trouble.

Thank you very much!
